artifactory.py

```diff
@@ -640,7 +640,13 @@ def __iter__(self):
         return self.iterator
 
 
-class _ArtifactoryAccessor(pathlib._Accessor):
+_artifactory_access_parent_class = (
+    pathlib.WindowsPath if os.name == "nt" else pathlib.PosixPath
+)
+
+
+class _ArtifactoryAccessor(_artifactory_access_parent_class):
```

Comment:
Do we need to have a base class for `_ArtifactoryAccessor`? 
`ArtifactoryPath` is a [Path itself](https://github.com/devopshq/artifactory/blob/master/artifactory.py#L1459) and it'll have accessor with Path class too - looks a bit weird

Reply:
What if we just remove it and make it `class _ArtifactoryAccessor`

Reply:
Original `pathlib._Accessor` didn't do too much :) 

https://github.com/python/cpython/pull/25701/files#diff-fa525485738fc33d05b06c159172ff1f319c26e88d8c6bb39f7dbaae4dc4105cL278-L280

Reply:
As you said, the base class for _ArtifactoryAccessor can be removed.